packaging/cellcc-el7.spec

```diff
@@ -65,5 +65,65 @@ mkdir -p %{buildroot}/%{_var}/lib/cellcc/restore-scratch
 %doc doc/example_remctl.conf
 
 %changelog
+
+* Mon Dec 07 2020 Andrew Deason <adeason@sinenomine.net> - v1.21
```

Comment:
hmm, when we spoke earlier and you mentioned putting 'git log' data into the specfile changelog, I thought we were talking about generating the %changelog data as part of the build process. I definitely do not want to keep 'git log' data checked in to a file like this.

but if you wanted to have something in './packaging/rules srpm7' generate this at build time, that would be okay. that is, we keep a cellcc-el7.spec.in file, and run some script like './generate-changelog cellcc-el7.spec.in > cellcc-el7.spec' (the .spec.in file is in git, but the .spec file is not). or if it's possible for the %prep or %build steps in the specfile to do that, that's another way (but I assume it's not possible to alter a spec file during those steps?)